src/components/textField/ValidationMessage.tsx

```diff
@@ -39,7 +39,7 @@ const ValidationMessage = ({
   if (validationIcon?.source) {
     return (
       <View row>
-        {showValidationMessage &&
+        {showValidationMessage && validationMessage &&
```

Comment:
Isn't it better to change `showValidationMessage`?
Either way, it should be `!!validationMessage`.